modules/Core/module.php

```diff
@@ -923,14 +923,7 @@ public function onPageLoad(User $user, Pages $pages, Cache $cache, Smarty $smart
                         $full_ip = ['ip' => $default->ip . (is_null($default->port) ? '' : ':' . $default->port), 'pre' => $default->pre, 'name' => $default->name];
 
                         // Get query type
-                        $query_type = Util::getSetting('external_query');
-                        if ($query_type == QueryType::INTERNAL) {
-                            $query_type = 'internal';
-                        } else if ($query_type == QueryType::EXTERNAL) {
-                            $query_type = 'external';
-                        } else if ($query_type == QueryType::PLUGIN) {
-                            $query_type = 'plugin';
-                        }
+                        $query_type = Util::getSetting('query_type', 'internal');
```

Comment:
We can remove this fallback, since we have it in the DatabaseInitialiser and the upgrade script we can safely assume it exists

Reply:
Isn't it always safe to have a fallback just in case?